### ISSUE TITLE
Feature: colour settings

### ISSUE THREAD

**Issue by sfkpmr** [pi-hole/AdminLTE#1250]
**In raising this issue, I confirm the following:** `{please fill the checkboxes, e.g: [X]}`

- [x] I have read and understood the [contributors guide](https://github.com/pi-hole/pi-hole/blob/master/CONTRIBUTING.md).
- [x] The issue I am reporting can be *replicated*.
- [x] The issue I am reporting isn't a duplicate (see [FAQs](https://github.com/pi-hole/pi-hole/wiki/FAQs), [closed issues](https://github.com/pi-hole/pi-hole/issues?utf8=%E2%9C%93&q=is%3Aissue%20is%3Aclosed%20), and [open issues](https://github.com/pi-hole/pi-hole/issues)).

The dark colour below feels very depressing. It would be nice to be able to pick graph colours.

![image](https://user-images.githubusercontent.com/10453979/80484887-bf6de880-8958-11ea-8ef6-e3b8a5a6148d.png)

**Comment by yubiuser** [comment]
@dschaper 

Do you think this should stay open? I labeled accessibility issues with `bug` to prevent them from going stale. But this seems more like a feature request for other reasons. The link posted by the OP already links a discourse feature request.

**Comment by dschaper** [comment]
I'd like to keep it open. A13Y could be the label if you'd like or just internal. This would be a good candidate for a test of the GitHub Projects tracking.

**Comment by rdwebdesign** [comment]
I think this is a request for color customization, not A13Y.
> It would be nice to be able to pick graph colours.

**Comment by dschaper** [comment]
Yeah, the color choices do like with color-blindness and a13y though. But I see your point.

**Comment by DL6ER** [comment]
The v6 rewrite added multiple themes including color-blind-friendly options, which covers the accessibility angle. Custom per-user color picking for graphs remains a feature request with no clear demand beyond the original reporter.

This has been open for 6 years with no implementation progress. The Discourse feature request (linked in comments) is the appropriate place to track community interest. Closing — can be reopened if there's renewed demand but I think this is covered.
